src/HotChocolate/Core/src/Execution/DependencyInjection/RequestExecutorServiceCollectionExtensions.cs

```diff
@@ -163,15 +163,6 @@ private static IRequestExecutorBuilder CreateBuilder(
 
         builder.Services.AddValidation(schemaName);
 
-        builder.Configure(
```

Comment:
This should no longer be necessary, as the cache will go out of scope as soon as the services of the evicted executor are disposed.